### PR TITLE
PWGHF: Add Bs candidate creator, selector, and task for reduced data format

### PR DESCRIPTION
**Pull request by fcatalan92**
@fchinu @AlexBigO @apalasciano I made a couple of simplifications and fixes also for B0 and B+, please have a look

### CODE REVIEW

**Comment by apalasciano** [comment]
Ciao @fcatalan92, thanks for propagating the changes in the other workflows as well.
They are fine to me

**Comment by fchinu** [comment]
Ciao @fcatalan92, thanks a lot for your developments, and for also modifying the other workflows as well! 
Everything seems fine also to me!

File: PWGHF/D2H/Utils/utilsRedDataFormat.h [hunk 1]
```diff
@@ -51,4 +51,31 @@ void checkEvSel(Coll const& collision, o2::hf_evsel::HfEventSelection& hfEvSel,
...
+{
+  float defaultNSigma = -999.f; // -999.f is the default value set in TPCPIDResponse.h and PIDTOF.h
+
+  bool hasTpc = prong1.hasTPC();
+  bool hasTof = prong1.hasTOF();
+
+  if (hasTpc && hasTof) {
+    float tpcNSigma = prong1.tpcNSigmaPi();
+    float tofNSigma = prong1.tofNSigmaPi();
+    return sqrt(.5f * tpcNSigma * tpcNSigma + .5f * tofNSigma * tofNSigma);
```
Comment (vkucera) [inline PWGHF/D2H/Utils/utilsRedDataFormat.h:69]:
```suggestion
    return std::sqrt(.5f * tpcNSigma * tpcNSigma + .5f * tofNSigma * tofNSigma);
```

File: PWGHF/D2H/Utils/utilsRedDataFormat.h [hunk 2]
```diff
@@ -51,4 +51,31 @@ void checkEvSel(Coll const& collision, o2::hf_evsel::HfEventSelection& hfEvSel,
...
+  }
+  if (hasTpc) {
+    return abs(prong1.tpcNSigmaPi());
```
Comment (vkucera) [inline PWGHF/D2H/Utils/utilsRedDataFormat.h:72]:
```suggestion
    return std::abs(prong1.tpcNSigmaPi());
```
`abs` returns `int`!

File: PWGHF/D2H/Utils/utilsRedDataFormat.h [hunk 3]
```diff
@@ -51,4 +51,31 @@ void checkEvSel(Coll const& collision, o2::hf_evsel::HfEventSelection& hfEvSel,
...
+  }
+  if (hasTof) {
+    return abs(prong1.tofNSigmaPi());
```
Comment (vkucera) [inline PWGHF/D2H/Utils/utilsRedDataFormat.h:75]:
```suggestion
    return std::abs(prong1.tofNSigmaPi());
```

**Review by vkucera — CHANGES_REQUESTED** [review]

**Comment by fcatalan92** [comment]
@vkucera Thanks for spotting this, addressed!

**Review by vkucera — APPROVED** [review]

**Pull request merged**